express/code/features/table-of-contents-seo/table-of-contents-seo.css

```diff
@@ -175,8 +177,50 @@ main .section .mobile-toc {
     transform: rotate(135deg);
 }
 
+@media (max-width: 599px) {
```

Comment:
I don't think you need max-width definition here, since the default style is everything, and then min-width down below fixes that.